### PR TITLE
feat(#552): Adding global config artifactsToSource

### PR DESCRIPTION
**Pull request by alexander-matthiesen**
Refers to #552

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,6 +249,11 @@ produce:
   artifacts: { paths: [path/] }
 ```
 
+A global configuration is possible when setting the following flag
+```shell
+gitlab-ci-local --no-artifacts-to-Source
```
Comment (firecow) [inline README.md:254]: Lower case S

**Review by firecow — APPROVED** [review]
Epic stuff, I've invited you to be collaborator 😁

File: README.md (hunk 1 above)
Comment (alexander-matthiesen) [reply README.md:254]:
```suggestion
gitlab-ci-local --no-artifacts-to-source
```

**Comment by firecow** [comment]
@alexander-matthiesen Ready to merge?

**Pull request merged**